app/src/main/java/com/supertokens/session/FrontToken.java

```diff
@@ -120,6 +120,12 @@ private static void removeTokenFromStorage(Context context) {
     public static void removeToken(Context context) {
         synchronized (tokenLock) {
             removeTokenFromStorage(context);
+            // We are clearing all stored tokens here, because:
+            // 1. removing FrontToken signals that the session is being cleared
+            // 2. you can only have a single active session - this means that all tokens can be cleared from all auth-modes if one is being cleared
+            // 3. some proxies remove the empty headers used to clear the other tokens (i.e.: https://github.com/supertokens/supertokens-website/issues/218)
+            Utils.setToken(Utils.TokenType.ACCESS, "", context);
+            Utils.setToken(Utils.TokenType.REFRESH, "", context);
```

Comment:
these will only clear storage releated to header based auth? or even cookie based?